docs/instruments/ZJSONInstrument_v0.0.1.md

```diff
@@ -0,0 +1,1028 @@
+together and is separated by a delimiter.
+
+They often have rules which work together and may be interdependent. Groups have the following form:
+
+```js
+{
+    "schema" : {
+        ...
+        "helpers" : {
+            "nameOfHelper" : {
+                "type": "group",
+                "description": {
+                    ...
+                },
+                "options": {
+                    "order": [
+                        string,
+                        string,
+                        ...
+                    ]
+                },
+                "rules": {
+                    ...
+                }
+        ...
+}
+```
+
+`options.order`: Required array of strings.
+                 An array of the names of `fields` or `helpers`, in order, that are to be formatted
+                 within this group. It may contain any field, but only helpers of the type "static".
+ 
+### 4.1.3: Row
+
+A helper of type "row" denotes a row in a table. If rows are not in a table, they should be rendered
+as a "group" helper.
+
+```js
+{
+    "schema" : {
+        ...
+        "helpers" : {
+            "nameOfHelper" : {
+                "type": "row",
+                "description": {
+                    ...
+                },
+                "options": {
+                    "order": [
+                        string,
+                        string,
+                        ...
+                    ]
+                },
+                "rules": {
+                    ...
+                }
+        ...
+}
+```
+
+`options.order`: Required array of strings.
+                 An array of the names of `fields` or `helpers`, in order, that are to be
+                 formatted within the row. It may contain any field, but only helpers of the type "static" and "group".
+
+The size of this array must equal the `options.colSize` value of the table to which the row belongs.
+
+### 4.1.4: Table
+
+A helper of type "table" denotes a list of "row" helpers that are displayed together in a tabular
+form.
+
+A table contains only "row" helpers whose `options.order` array must be of the same size.
+
+```js
+{
+    "schema" : {
+        ...
+        "helpers" : {
+            "nameOfHelper" : {
+                "type": "table",
+                "description": {
+                    ...
+                },
+                "options": {
+                    "colSize" : int,
+                    "rowSize" : int,
+                    "order": [
+                        string,
+                        string,
+                        ...
+                    ],
+                    "showColHeaders" : boolean,
+                    "showDesc" : boolean,
+                    "showAsFooter": boolean,
+                },
+                "rules": {
+                    ...
+                }
+        ...
+}
+```
+`options.colSize` : Required int.
+                    The number of columns in the table. 
```

Comment:
Can't this be inferred?

Reply:
this is also for validating the elements of the table